pkg/console/subresource/deployment/deployment.go

```diff
@@ -171,17 +180,26 @@ func withConsoleAnnotations(
 	oAuthClientSecret *corev1.Secret,
 	proxyConfig *configv1.Proxy,
 	infrastructureConfig *configv1.Infrastructure,
+	authServerCAConfigMap *corev1.ConfigMap,
```

Comment:
it should now be possible to unify `authServerCAConfigMap` and `oauthServingCertConfigMap` here

Reply:
Handled in #842.